eng/build.ps1

```diff
@@ -14,25 +14,43 @@ Param(
   [Parameter(ValueFromRemainingArguments=$true)][String[]]$properties
 )
 function Print-Usage() {
-    Write-Host "Default if no actions are passed in: -restore -build"
-    Write-Host ""
-    Write-Host "CoreFx specific actions:"
-    Write-Host "  -buildtests             Build test projects in the solution"
-    Write-Host "  -clean                  Clean the solution"
-    Write-Host ""
-    Write-Host "CoreFx specific options:"
-    Write-Host "  -framework              The target group assemblies are built for (short: -f)"
-    Write-Host "  -os                     The operating system assemblies are built for"
-    Write-Host "  -allconfigurations      Build packages for all build configurations"
-    Write-Host "  -coverage               Collect code coverage when testing"
-    Write-Host "  -outerloop              Include tests which are marked as OuterLoop"
-    Write-Host "  -arch                   The architecture group (x86, x64, arm, etc.)"
-    Write-Host ""
+  Write-Host "Common settings:"
+  Write-Host "  -framework              The target group assemblies are built for (short: -f)"
+  Write-Host "  -configuration <value>  Build configuration: 'Debug' or 'Release' (short: -c)"
+  Write-Host "  -verbosity <value>      Msbuild verbosity: q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic] (short: -v)"
+  Write-Host "  -binaryLog              Output binary log (short: -bl)"
+  Write-Host "  -help                   Print help and exit"
+  Write-Host ""
+
+  Write-Host "Actions (defaults to -restore -build):"
+  Write-Host "  -restore                Restore dependencies (short: -r)"
+  Write-Host "  -build                  Build solution (short: -b)"
+  Write-Host "  -buildtests             Build test projects in the solution"
+  Write-Host "  -rebuild                Rebuild solution"
+  Write-Host "  -test                   Run all unit tests in the solution (short: -t)"
+  Write-Host "  -performanceTest        Run all performance tests in the solution"
+  Write-Host "  -pack                   Package build outputs into NuGet packages"
+  Write-Host "  -sign                   Sign build outputs"
+  Write-Host "  -publish                Publish artifacts (e.g. symbols)"
+  Write-Host "  -clean                  Clean the solution"
+  Write-Host ""
+
+  Write-Host "Advanced settings:"
+  Write-Host "  -coverage               Collect code coverage when testing"
+  Write-Host "  -outerloop              Include tests which are marked as OuterLoop"
+  Write-Host "  -allconfigurations      Build packages for all build configurations"
+  Write-Host "  -os                     The operating system assemblies are built for"
```

Comment:
Include examples, as with "arch"?

Reply:
And arch is missing arm64(?

For all these that are "enumerations" please include all the possible values or the everyday ones. I otherwise have to guess or read the scripts sometimes. I'm sure that's true for new users